docs/section4/4-3.md

```diff
@@ -0,0 +1,46 @@
+# ASHVAC – Rule 4-3
+
+**Rule ID:** 4-3
+ 
+**Rule Description:** HVAC Fan Schedules. Schedules for HVAC fans that provide outdoor air for ventilation shall run continuously whenever spaces are occupied.  
+**Rule Assertion:** P-RMR = expected value                                           
+**Appendix G Section:** Section G3.1-4 Schedules for the proposed building  
+**Appendix G Section Reference:**  
+
+**Applicability:** All required data elements exist for P_RMR  
+**Applicability Checks:** 
+
+1. Not applicable for for heating or cooling system is being simulated only to meet the requirements described in Table G3.1.
+
+**Manual Check:** None  
+**Evaluation Context:** Evaluate each data element  
+**Data Lookup:** None  
+**Function Call:** None
+
+## Rule Logic:
+- For each HVAC system in P_RMR: `for hvac_p in P_RMR...heating_ventilation_air_conditioning_systems:`
+    - Get HVAC system status: `status_type_p = hvac_p.status_type:`
+        - Check if HVAC system status is new or existing: `if status_type_p in [NEW,EXISTING]:`
```

Comment:
I need an explanation on this from the group/yuo on this. I am having trouble understanding how the logic knows this. I see that Xing has included code to determine the applicability so that is what I did. I think exception #1 applies to the status because if its a yet to be designed system then #1 would come into play.